src/containers/Ideas/ExplorePage.js

```diff
@@ -15,10 +15,37 @@ import { ExploreListItem } from './ExploreListItem'
 import { styles } from './ExplorePage.styles'
 
 export const ExplorePage = (props) => {
-  const [user, setUser] = useState()
   const [projects, setProjects] = useState([])
   const {signOut} = useAuthorization()
-  const { deepLink, setDeepLink } = useAuthorization()
+  const { user, deepLink, setDeepLink } = useAuthorization()
+
+  useFocusEffect(
```

Comment:
I just moved this to the top (as hooks are not called I find it better if they are at the top where you can see them)